components/network/asterisk/Makefile

```diff
@@ -26,7 +27,9 @@ COMPONENT_ARCHIVE_URL=	http://downloads.asterisk.org/pub/telephony/asterisk/old-
 COMPONENT_LICENSE=	GPLv2
 COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
 
-include $(WS_MAKE_RULES)/common.mk
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
 
```

Comment:
You changed from the new style of imports to the old one. Please revert that and add TEST_TARGET like so:
TEST_TARGET= $(NO_TESTS)
include $(WS_MAKE_RULES)/common.mk